### PR TITLE
Move case study to top of optimizations document and add link to best practices doc

### PR DESCRIPTION
**Pull request by teesloane**
Requested changes from Will. 

- move the Coinbase link to the top of the document
- add a link to a Showpad document on resource class best practices.

### CODE REVIEW

File: jekyll/_cci2/optimizations.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ This document provides an overview of several methods for optimizing your Circle
 * TOC
 {:toc}
 
+**Note**: If you are interested in reading about configuration optimizations at a large scale, consider reviewing Coinbase's article [Continuous Integration at Coinbase: How we optimized CircleCI for speed and cut our build times by 75%](https://blog.coinbase.com/continuous-integration-at-coinbase-how-we-optimized-circleci-for-speed-cut-our-build-times-by-378c8b1d7161).
```
Comment (michelle-luna) [inline jekyll/_cci2/optimizations.md]: Leave this at the bottom of the page please.

File: jekyll/_cci2/optimizations.md [hunk 2]
```diff
@@ -136,6 +138,8 @@ jobs:
       - run: make
 ```
 
+"For more details on how to best use resource classes to speed up build times, consider reading the [Credit Usage Examples & Best Practices](https://circleci.showpad.com/share/fwxM6b3HTPEHUG4AvnTCI) document.
```
Comment (michelle-luna) [inline jekyll/_cci2/optimizations.md]:
```suggestion
For more details on how to best use resource classes to speed up build times, consider reading the [Credit Usage Examples & Best Practices](https://circleci.showpad.com/share/fwxM6b3HTPEHUG4AvnTCI) document.
```

**Comment by teesloane** [comment]
It does seems a bit marketing heavy / ie belongs at the bottom of the docs; I think I will pass this info back.

**Comment by michelle-luna** [comment]
oh, right and the showpad link, this is really a strange way to do this. I'm going to close this for now, Tyler if you get more requests, please just route them to me, so I can get the gist before we prioritize it.

**Pull request closed without merging**